Localize/lang/strings.json

```
@@ -2199,6 +2191,7 @@
   "gMhc5o": "Required. The string to combine into a single string.",
   "gQt/0f": "Conversion functions",
   "gRUmiA": "Info",
+  "gRyFIu": "Copy value of {label} to clipboard",
   "gTXyUo": "Time Zone",
   "gUp23R": "The name already exists or is invalid, please update it before proceeding.",
```

"Please" is unnecessary.

```suggestion
  "gUp23R": "The name already exists or is invalid. Update the name before you continue.",
```